tutorials/lava/lib/peripherals/dvs/PropheseeCamera.ipynb

```diff
@@ -0,0 +1,313 @@
+   "source": [
+    "filters = [TrailFilterAlgorithm(width=width, height=height, threshold=100000),\n",
+    "           ActivityNoiseFilterAlgorithm(width=width, height=height, threshold=1000),]\n",
+    "\n",
+    "\n",
+    "# Init Processes\n",
+    "frame_input = PropheseeCamera(device=SEQUENCE_FILENAME_RAW,\n",
+    "                              filters=filters,\n",
+    "                              sensor_shape=(height, width),\n",
+    "                              num_output_time_bins=1)\n",
+    "\n",
+    "recv = VisProcess(shape=frame_input.s_out.shape)\n",
+    "\n",
+    "# Connect\n",
+    "frame_input.s_out.connect(recv.s_in)\n",
+    "\n",
+    "# Run\n",
+    "num_steps = 200\n",
+    "run_cfg = Loihi2SimCfg()\n",
+    "run_cnd = RunSteps(num_steps=num_steps)\n",
+    "\n",
+    "frame_input.run(condition=run_cnd, run_cfg=run_cfg)\n",
+    "frame_input.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebade859-eaca-4b07-a6df-3ecf4136df62",
+   "metadata": {},
+   "source": [
+    "## Apply transformations\n",
+    "\n",
+    "We provide numerous transformations to be applied to the event data. For example Downsampling, MergePolarities and Mirroring."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e91714a-baa6-4dfb-aa62-af0932e017d7",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<table>\n",
+    "<tr>\n",
+    "    <td> <img src=\"gifs/transform.gif\" alt=\"Drawing\" style=\"height: 125px;\"/> </td>\n",
+    "</tr>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "3c87662b-1fe1-4594-bcb0-2a30da9bf41a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transformations = Compose(\n",
+    "    [\n",
+    "        Downsample(factor=0.5),\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "# Init Processes\n",
+    "frame_input = PropheseeCamera(device=SEQUENCE_FILENAME_RAW,\n",
+    "                              transformations=transformations,\n",
+    "                              sensor_shape=(height, width),\n",
+    "                              num_output_time_bins=1)\n",
+    "\n",
+    "recv = VisProcess(shape=frame_input.s_out.shape)\n",
+    "\n",
+    "# Connect\n",
+    "frame_input.s_out.connect(recv.s_in)\n",
+    "\n",
+    "# Run\n",
+    "num_steps = 200\n",
+    "run_cfg = Loihi2SimCfg()\n",
+    "run_cnd = RunSteps(num_steps=num_steps)\n",
+    "\n",
+    "frame_input.run(condition=run_cnd, run_cfg=run_cfg)\n",
+    "frame_input.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "66130c3c-8e25-46ae-8433-2c3a2853027b",
+   "metadata": {},
+   "source": [
+    "## Custom transformation and manual output shape\n",
+    "\n",
+    "We also provide the option to use tonic transformations on the data. See the tonic documentation to see a complete list and usage. If you need to use custom transformation, the automatic shape determination would not work. In that case, you can specify the output shape manually."
```

Comment:
This could more explicitly state that the transformations are compatible with both transformations from `tonic` and `pytorch`.